features/i18n.feature

```diff
@@ -12,35 +12,30 @@ Feature: I18n
           Behat\Behat\Tester\Exception\PendingException;
       use Behat\Gherkin\Node\PyStringNode,
           Behat\Gherkin\Node\TableNode;
+      use Behat\Step\Given;
+      use Behat\Step\Then;
+      use Behat\Step\When;
 
       class FeatureContext implements Context
       {
           private $value = 0;
 
-          /**
-           * @Given /Я ввел (\d+)/
-           */
+          #[Given('/Я ввел (\d+)/')]
```

Comment:
I'm not certain, but it *might* be worth at least one simple test to prove that the russian is also translated to match e.g. the `@Given` annotation? I think at the moment the parsing/i18n is implemented so that this happens before it looks for definitions (so we can be pretty sure that annotations and attributes will behave the same), but there's no guarantee that would stay the same in future.